### PR TITLE
Improve documentation for CodeInstruction

### PR DESCRIPTION
**Pull request by asquared31415**
- Minor spelling/grammar fixes
 - Elaboration on the creation of labels
 - Warn users to not use `ILGenerator.Emit()`

### CODE REVIEW

File: Harmony/Documentation/articles/patching-transpiler-codes.md [hunk 1]
```diff
@@ -24,15 +24,17 @@ Some things though will be restricted:
 - `SignatureHelper` support is experimental at best
 - You should avoid using indices when referring to local variables
 
-In general, it is advinced to reuse and to copy existing operands for things like labels and local variables. Search for a significant and unique location in the existing codes and grab the operand from there. This will allow you to refer to local variables and labels in a change-resistant way.
+Do **not** use `ILGenerator.Emit()`. While it does create IL code, Harmony is an abstraction over it. Harmony takes the returned `IEnumerable<CodeInstruction>` and after some manipulation, calls `Emit()` itself. `ILGenerator` should only be used for its methods to define blocks and labels. See [Labels](#labels) and [Try/catch boundaries](#trycatch-boundaries) for more information.
```
Comment (asquared31415) [inline Harmony/Documentation/articles/patching-transpiler-codes.md:27]: I was unable to test whether these links to specific headers worked, but I believe they should.

Comment (pardeike) [reply Harmony/Documentation/articles/patching-transpiler-codes.md:27]: Ah. I *think* it’s just relative to the current directory and you name the whole file including the extension. Let’s do this like this: I’ll merge and if anything does not look right you let me know and I’ll fix it or you make just another PR. 

Thanks for your contribution!

**Comment by asquared31415** [comment]
It does work, and it appears the links get just directly set to the `href` in an `<a>`, when the docs are compiled so all the weirdness from various URL schemes works.

**Comment by pardeike** [comment]
Nice

**Pull request merged**